react-function-components/src/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function CustomButton() {
+  return <button>Click Me!</button>;
+}
+
+const container = document.querySelector('#root');
+const root = ReactDOM.createRoot(container);
+
+root.render(CustomButton());
```

Comment:
We never call a component directly. Instead we use JSX:
```JSX
root.render(<CustomButton />);
```

Reply:
gotcha! used JSX instead!